dtos/requests/deviceprofiletags.go

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright (C) 2025 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package requests
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v4/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v4/dtos"
+	dtoCommon "github.com/edgexfoundry/go-mod-core-contracts/v4/dtos/common"
+	edgexErrors "github.com/edgexfoundry/go-mod-core-contracts/v4/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v4/models"
+)
+
+// DeviceProfileTagsRequest defines the Request Content for PATCH UpdateDeviceProfileTags DTO.
+type DeviceProfileTagsRequest struct {
+	dtoCommon.BaseRequest        `json:",inline"`
+	dtos.UpdateDeviceProfileTags `json:",inline"`
+}
+
+// Validate satisfies the Validator interface
+func (dt DeviceProfileTagsRequest) Validate() error {
```

Comment:
To align with UnmarshalJSON func, use pointer receiver here.
```suggestion
func (dt *DeviceProfileTagsRequest) Validate() error {
```